src/event-listeners/onDocumentLoaded.js

```diff
@@ -44,8 +44,18 @@ export default dispatch => () => {
     dispatch(actions.setOutlines(outlines));
   });
 
+  const doc = core.getDocument();
+  doc.getLayersArray().then(layers => {
+    if (layers.length === 0) {
+      dispatch(actions.disableElement('layersPanel', 1));
+      dispatch(actions.disableElement('layersPanelButton', 1));
+    } else {
+      dispatch(actions.setLayers(layers));
```

Comment:
We should enable layersPanel and layersPanelButton here for subsequent documents